### ISSUE TITLE
Theme status?

### ISSUE THREAD

**Issue by deap82** [statiqdev/Samson#5]
What is the status of this theme?
I'm trying to replace a Wyam site with Statiq and this theme seems to be more close to what I need than the CleanBlog theme.
But when I try to use it I get build errors on usages of "CodeAnalysisKeys".

**Comment by deap82** [comment]
I've found the Statiq.Docs package now and is using CreateDocs instead of CreateWeb in my Main.

Had to change 
`Context.GetTypeLink(doc, Model.LinkTypeArguments)`
to 
`@IExecutionContextExtensions.GetTypeLink(Context, doc, Model.LinkTypeArguments)`
in _DocsTable.cshtml

to not get

_The call is ambiguous between the following methods or properties: 'Statiq.Docs.IExecutionContextExtensions.GetTypeLink(Statiq.Common.IExecutionContext, Statiq.Common.IDocument, bool)' and 'Generator.Extensions.GetTypeLink(Statiq.Common.IExecutionContext, Statiq.Common.IDocument, bool)'_

But generated site is still like using the CleanBlog theme. So question about theme status remains.

**Comment by daveaglick** [comment]
I actually need to delete this repository - porting the theme in-place proved too difficult and I kept stalling out so I started from scratch over in https://github.com/statiqdev/Docable and that's now the "official" Statiq Docs theme. It's coming along quite well and should be (mostly) usable:

![image](https://user-images.githubusercontent.com/1020407/156777747-c6009098-6365-44a7-bb5b-7faa99972c43.png)

I'm still working on a little more documentation and guidance before announcing it publicly, so keep in mind there may still be rough edges. Go ahead and try it out and let me know if you encounter any problems.
